database-grants/terraform/dev/schemas/queue/terraform.tfstate

```diff
@@ -0,0 +1,99 @@
+{
+  "version": 4,
+  "terraform_version": "1.7.5",
+  "serial": 48,
+  "lineage": "ee067647-f617-042b-eb7a-124fc2deb440",
+  "outputs": {},
+  "resources": [
+    {
+      "mode": "managed",
+      "type": "random_password",
+      "name": "s_password",
+      "provider": "provider[\"registry.terraform.io/hashicorp/random\"]",
+      "instances": [
+        {
+          "schema_version": 3,
+          "attributes": {
+            "bcrypt_hash": "$2a$10$odW0epAAbhsigKjmtMBXcueVuXIZL8.fdDNOUcdrTAZ7o1pRRtkRW",
+            "id": "none",
+            "keepers": null,
+            "length": 16,
+            "lower": true,
+            "min_lower": 0,
+            "min_numeric": 0,
+            "min_special": 0,
+            "min_upper": 0,
+            "number": true,
+            "numeric": true,
+            "override_special": "_%@",
+            "result": "POda9YkErd@9Ngbv",
+            "special": true,
+            "upper": true
+          },
+          "sensitive_attributes": []
+        }
+      ]
+    },
+    {
+      "module": "module.microservice_schema",
+      "mode": "managed",
+      "type": "random_password",
+      "name": "m_password",
+      "provider": "provider[\"registry.terraform.io/hashicorp/random\"]",
+      "instances": [
+        {
+          "schema_version": 3,
+          "attributes": {
+            "bcrypt_hash": "$2a$10$ajk0Zr7pifvnvGP7DfwbgeaaKEq8JdrsyNlQiKHXrdvTdndEyRRn2",
+            "id": "none",
+            "keepers": null,
+            "length": 16,
+            "lower": true,
+            "min_lower": 0,
+            "min_numeric": 0,
+            "min_special": 0,
+            "min_upper": 0,
+            "number": true,
+            "numeric": true,
+            "override_special": "_%@",
+            "result": "v813@rRhHCwhJMs8",
```

Comment:
@partha04patel why do we push the terraform state to github? is this the password?